src/Slider.lua

```diff
@@ -171,8 +179,17 @@ function Slider:render()
 				end
 			end,
 		}, {
+			Background = Roact.createElement(
+				self.props.Background,
+				{
+					Disabled = props.Disabled,
+					Hover = self.state.Hover,
+					Dragging = self.state.Dragging,
+					Value = self.props.Value,
+				}
+			),
```

Comment:
We need to prevent a user's custom Background overlapping the slider bar/handle if they used a ZIndex greater than 0. This wraps it in a holder frame at the correct ZIndex:

```suggestion
			BackgroundHolder = Roact.createElement("Frame", {
				ZIndex = 0,
				Size = UDim2.fromScale(1, 1),
				BackgroundTransparency = 1,
			}, {
				Background = Roact.createElement(self.props.Background, {
					Disabled = props.Disabled,
					Hover = self.state.Hover,
					Dragging = self.state.Dragging,
					Value = self.props.Value,
				}),
			}),
```

Reply:
Good idea!